opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipOverview.jsx

```diff
@@ -211,6 +212,7 @@ class StixSightingRelationshipContainer extends Component {
       classes,
       stixSightingRelationship,
       paddingRight,
+      isFABReplaced = false,
```

Comment:
I thought first "not ideal, we should refactor this file to use hooks"
But then I looked at the file size.

... yeah let's use a temporary prop, until we remove the feature flag.